examples/parallelplot.py

```diff
@@ -23,4 +23,7 @@
     obs = pm.Normal('obs', mu=theta, sd=sigma, observed=y)
     centered_eight_trace = pm.sample()
 
-az.parallelplot(centered_eight_trace, varnames=['theta', 'tau', 'mu'])
+az.parallelplot(centered_eight_trace, var_names=['theta', 'tau', 'mu'])
+
+import matplotlib.pyplot as plt
```

Comment:
are these last two lines supposed to be here?

Reply:
Just for WIP. I'll take it out when I replace this commit.